fcq/views.py

```diff
@@ -0,0 +1,56 @@
+import operator
+import re
+from functools import reduce
+
+from django.db.models import Q
+from django.http import JsonResponse
+from django.shortcuts import render
+from django.views.generic import CreateView, DetailView, ListView, UpdateView
+from django.db.models.functions import Substr, Lower
+
+from fcq.models import Teacher
+from rest_framework import serializers
+from .serializers import TeacherSerializer
+
+
+# Create your views here.
+
+def fcq_search(request):
+    return render(request, "fcq/fcq_search.html")
+
+def fcq_display(request):
+        return render(request, "fcq/fcq_display.html")
+
+def fcq_search_ajax(request):
```

Comment:
Search feature works fine, it would be nice to get the page to not reload when I submit a new search though

Reply:
I second this. It should be done using an ajax call and populated with JavaScript. Is there a reason why you changed the view on search? I would keep the form the same regardless if its the first or sequential searches